From e6a9c366ac51100f8516002fdef05ef206135bee Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Wed, 27 Feb 2019 21:24:16 +0200
Subject: [PATCH] CalendarProvider: Remove unused string resource

The original string was removed in commit a48a040eaad1bc32d79b31d36fb68003f768ba6e
but translations for this locale were never updated

Fixes:
warning: string 'upgrade_msg' has no default translation.
warning: string 'upgrade_msg' is missing 87 required localizations:
af_ZA am_ET ar_EG as_IN ast_ES be_BY bg_BG bn_BD bs_BA ca_ES cs_CZ cy_GB da_DK
de_DE el_GR en_AU en_CA en_GB en_IN en_US en_XC es_ES es_US et_EE eu_ES fa_IR
fi_FI fr_CA fr_FR gd_GB gl_ES gu_IN hi_IN hr_HR hu_HU hy_AM in_ID is_IS it_IT
iw_IL ja_JP ka_GE kk_KZ km_KH kn_IN ko_KR ky_KG lo_LA lt_LT lv_LV mk_MK ml_IN
mn_MN mr_IN ms_MY my_MM nb_NO ne_NP nl_NL or_IN pa_IN pl_PL pt_BR pt_PT ro_RO
ru_RU si_LK sk_SK sl_SI sq_AL sr_Latn_RS sr_RS sv_SE sw_TZ ta_IN te_IN th_TH
tl_PH tr_TR uk_UA ur_PK uz_UZ vi_VN zh_CN zh_HK zh_TW zu_ZA

Change-Id: Ie8272c9697febfc24b4dabc5ec6fb3759d3bcffb
---
 res/values-az/strings.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/res/values-az/strings.xml b/res/values-az/strings.xml
index bb77a27..dd8c277 100644
--- a/res/values-az/strings.xml
+++ b/res/values-az/strings.xml
@@ -17,7 +17,6 @@
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
     <string name="calendar_storage" msgid="5387668002987562770">"Təqvim Yaddaşı"</string>
-    <string name="upgrade_msg" msgid="2792831029435070926">"Təqvim bazası təkmilləşdirilir."</string>
     <string name="calendar_default_name" msgid="6924293766625167275">"Defolt"</string>
     <string name="calendar_info" msgid="6687678621418059281">"Təqvim məlumatı"</string>
     <string name="calendar_info_error" msgid="5575162446528419982">"Xəta"</string>
-- 
2.17.1

